examples/helm-template/README.md

```diff
@@ -0,0 +1,36 @@
+# Helm Template
```

Comment:
How's this meshing with this doc:

https://googlecontainertools.github.io/kpt/guides/consumer/function/catalog/docs/helm-template/usage/

Reply:
Will integrate in a separate docs PR to kpt repo.